frontend/src/components/ConfirmationToast/ConfirmationToast.tsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
-import { check, block } from '../../icons/other/index';
+import React, { useState } from 'react';
+import { green_check, block, close } from '../../icons/other';
 import styles from './confirmationtoast.module.css';
 import { ToastStatus } from '../../context/toastContext';
 
 type toastProps = {
   message: string;
   toastType?: ToastStatus;
+  onClose: () => void;
 };
 
-const Toast = ({ message, toastType }: toastProps) => {
+const Toast = ({ message, toastType, onClose }: toastProps) => {
   return typeof toastType === 'undefined' ||
     toastType == ToastStatus.SUCCESS ? (
     <div className={`${styles.toast} ${styles.successToast}`}>
-      <img alt="toast check" src={check} />
-      <p className={styles.toasttext}>{message}</p>
+      <button
+        onClick={() => {
```

Comment:
You can directly call onClose like this `onClick={onClose}`. Lets keep code consistency